annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastPackageResolver.java

```diff
@@ -0,0 +1,98 @@
+package com.exactpro.epfast.annotation.processing;
+
+import com.exactpro.epfast.annotations.FastPackage;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.PackageElement;
+import javax.lang.model.util.Elements;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FastPackageResolver {
```

Comment:
I was probably wrong trying to separate FastPackageResolver and FastPackageCreator. We need to merge them.

It basically needs just three public methods: add(Element fastType), addFastPackage(Element fastPackage) and Collection<FastPackageElement> getFastPackages().
In annotation processor you need to collect all @FastType elements, add them, then collect all @FastPackage elements, add them (to ensure empty packages are added), and get the list of packages. Then this list needs to be checked for duplicates.